tests/test_app_e2e.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import os
+from threading import Thread 
+
+import trello_items as item_store
+
+from unittest import mock
+from dotenv import load_dotenv, find_dotenv
+import app
+
+# from hypothesis import given
+# from hypothesis.strategies import integers
+from selenium import webdriver
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    # Setup the test boards
+    if item_store.BOARD_ID == "":
+        boards = item_store.get_boards()
+        item_store.BOARD_ID = next((board['id'] for board in boards if board['name'] == "TestBoard"), "")
+
+    # construct the new application
+    application = app.create_app() 
+    # start the app in its own thread.
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield app
+    
+    # Tear Down
+    thread.join(1)
+    item_store.BOARD_ID = ""
+
+@pytest.fixture(scope='module')
+def driver():
+    # path to your webdriver download
+    opts = webdriver.ChromeOptions()
+    # opts.add_argument('--headless')
+    
+    # path to your webdriver download
+    with webdriver.Chrome('C:/Users/pcoul/Documents/CS Dipolma/Module 3/chromedriver_win32/chromedriver', options=opts) as driver:
```

Comment:
This obviously won't work on anyone else's computer and isn't documented. Even needing chromedriver isn't documented.

You should:
- add instructions to the README saying chromedriver is necessary for the e2e tests to run, and that they should add it to the checkout folder
- replace this file path with just 'chromedriver'
- add chromedriver to the .gitignore file.

Reply:
Reworked